docs/Changelog.md

```diff
@@ -4,6 +4,26 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/)
 
+## <a name="4.0.0" />[4.0.0] - 2025-10-07
+
+See full log [of v3.11.0...v4.0.0](https://github.com/microsoft/testfx/compare/v3.11.0...v4.0.0)
+
+### Added
+
+* Support for `CallerArgumentExpression` in all Assert APIs for better messages of assertion failures.
+* Added `Assert.That`, which provides easy way to write any assertion boolean expression with clear error message in case of assertion failures.
+* Breaking changes: see [Migrate from MSTest v3 to MSTest v4](https://learn.microsoft.com/dotnet/core/testing/unit-testing-mstest-migration-v3-v4).
```

Comment:
It's a bit of an effort but I would expand the list here instead of only referring to the ticket.

Reply:
It's big IMO to be in the changelog.